### DISCUSSION TITLE
Install failed, chosen path then click install, program disappeared and not installed any files. （run with administrator account, failed to elevate privilege, can not run reg）

### DISCUSSION THREAD

**Discussion by wgf4242** [rustdesk/rustdesk#4494] [General]
rustdesk-1.1.9 in windows 10.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/releases/nightly
1.1.9 may have installation problem when the path contains special characters (non-english).
If nightly does not work, please paste your path here.

**Comment by wgf4242** [comment]
Tested 1.2.0,  video here.

https://github.com/rustdesk/rustdesk/assets/3336626/544b767e-8ff5-4e59-91f3-85c8d07f04f5

path: `C:\Program Files\RustDesk`

**Reply by rustdesk** [reply]
@21pages

**Comment by rustdesk** [comment]
Can you try out below? and show us the cmd window.

```
rustdesk.exe --silent-install 1
```

**Reply by wgf4242** [reply]
![bandicam 2023-05-31 00-04-37-195 mp4_000002 171](https://github.com/rustdesk/rustdesk/assets/3336626/0e338aee-16be-4221-b62e-19db18d7330c)

**Comment by rustdesk** [comment]
Can you run ` reg delete HKEY_CLASSES_ROOT\.rustdesk /f` manually? then show us the cmd window.

**Reply by wgf4242** [reply]
![bandicam 2023-05-31 11-20-28-186 mp4_000047 565](https://github.com/rustdesk/rustdesk/assets/3336626/59f6b52f-3514-40fe-9b99-f9350ad7cc2f)

**Reply by rustdesk** [reply]
Wired，the first one reported errors, but did not hang there, The second one did not report, but hang there.

**Reply by 21pages** [reply]
try this one, https://github.com/21pages/rustdesk/releases/download/nightly/rustdesk-1.2.0-x86_64-query-before-reg-delete.exe, if it's ok ,try reinstall again please. I'll provide one without reg delete later.

**Reply by 21pages** [reply]
https://github.com/21pages/rustdesk/releases/download/nightly/rustdesk-1.2.0-x86_64-without-reg-delete.exe

**Reply by wgf4242** [reply]
Thanks, successfully installed.
Files installed, but no service.

**Reply by rustdesk** [reply]
which one?

**Reply by rustdesk** [reply]
Your system's permission is very wierd.

**Reply by wgf4242** [reply]
https://github.com/21pages/rustdesk/releases/download/nightly/rustdesk-1.2.0-x86_64-without-reg-delete.exe

![bandicam 2023-05-31 15-23-37-219 mp4_000002 066](https://github.com/rustdesk/rustdesk/assets/3336626/a914459d-8214-4648-bfc8-796571d8b898)
![bandicam 2023-05-31 15-23-37-219 mp4_000001 999](https://github.com/rustdesk/rustdesk/assets/3336626/58b11ef8-cdb7-43ef-968b-858ae0b81791)

**Comment by rustdesk** [comment]


**Reply by 21pages** [reply]
Copy all the logs and paste.

**Comment by rustdesk** [comment]
The installation process stops at reg operation again.

**Comment by 21pages** [comment]
Try login with a user which username is not `Administrator`  and install.
